code/api/src/migrations/20210325161321-7-add_fields_to_users.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
```

Comment:
Alright, we switched to using `async` here. Which means the return of the `queryInterface.addColumn()` will be a `promise` object. Do you know why only some of the functions are async?

I see that it requires a different `queryInterface` command than when you're just `createtable` or `droptable` since instead of referring to the whole table, they're properties being added to the existing `users` table. Is it not possible too just edit the file that `create`s  the `users` in the first place and add these as a property? 

Why is it being done in this way? Is it because of signup only populating certain things or these not existing by default for every user unless added?

Reply:
`async` is auto-populated when we run a `generate:migration`. We switched most to `return` to stay consistent, but must have overlooked this. We can switch it to return if we run into issues! (: